src/Umbraco.Web.UI/Views/Partials/Grid/Editors/TextString.cshtml

```diff
@@ -4,10 +4,9 @@
 @if (Model.editor.config.markup != null)
 {
     string markup = Model.editor.config.markup.ToString();
-
     var UmbracoHelper = new UmbracoHelper(UmbracoContext.Current);
     
-    markup = markup.Replace("#value#", UmbracoHelper.ReplaceLineBreaksForHtml(Model.value.ToString()));
+    markup = markup.Replace("#value#", UmbracoHelper.ReplaceLineBreaksForHtml(TemplateUtilities.CleanForXss(Model.value.ToString())));
```

Comment:
I don't agree with this, querying the grid is ugly enough as it is. As you said, MVC will strip out the nasty stuff already so this is redundant and confusing.